sdk/python/pvsite_datamodel/read/generation.py

```diff
@@ -1,44 +1,40 @@
 """ Read pv generation functions """
 import logging
-from datetime import datetime, timezone
-from typing import List, Optional, Union
+import uuid
+from datetime import datetime
+from typing import List, Optional
 
-from sqlalchemy import desc
-from sqlalchemy.orm import Session, joinedload
+from pvsite_datamodel.sqlmodels import ClientSQL, DatetimeIntervalSQL, GenerationSQL, SiteSQL
+from sqlalchemy.orm import Session
 
-from pvsite_datamodel.sqlmodels import GenerationSQL, SiteSQL, DatetimeIntervalSQL, ClientSQL
+from .utils import filter_query_by_datetime_interval
```

Comment:
im never sure, but is it clear if it is `pvsite_datamodel.read.utils`?

Reply:
I don't know either - my instinct here was to have modules in the same package be allowed to use relative imports (since they're all at the same level and scoped to the package) with a small amount of guidance from https://peps.python.org/pep-0328/#rationale-for-relative-imports. The rationale being that this makes the internal package dependencies clearer and more seperate vs dependecy imports from a seperate package (but within the same repo). What do you prefer?

Reply:
Leave it how it is for the moment